src/models/Morph.js

```diff
@@ -0,0 +1,49 @@
+import { rotate, offset, scale } from '@/common/geometry'
```

Comment:
I was (and am) struggling with this:
 - Morph was how I was trying to describe something that can change size, change scale, and have an offset (translate). 
 - Shapes can't morph (they are kernels, they don't move or change size anymore).
 - I was trying to simplify these to only have one boolean canMorph. Because it felt just too complicated before. There is 1-2 shapes that previously can't change aspect ratio, IIRC. This is where I got stuck. So I am sort of one leg into morph and one leg still back in canChangeSize land.
 - The layer and the effect (and possibly the group) can "morph". I was trying to reuse that functionality in a common place (DRY).

I may have to give up and just keep all the canChangeWidth/etc flags. I probably still want to consider keeping all of that in a morph-like object. I want to call it "transform", because that's what I would call it in linear algebra, but transform already has a strong meaning in our code.